### PR TITLE
made a new function to print author of changes

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**

### CODE REVIEW

**Comment by mambelli** [comment]
I did not see any description, could you tell me what's this about?

**Comment by GuillermoFidalgo** [comment]
This PR is to add a new function that prints out the name of the author who made it.

File: src/planetsmath/functions.py [hunk 1]
```diff
@@ -28,3 +28,9 @@ def sum_product(list):
     for item in list:
         product *= item
     return product
+
+
+def make_author_print(author="Guillermo"):
+	"""This function makes a print of the author of this change""" 
+	print("Guillermo Made a fork")
```
Comment (mambelli) [inline src/planetsmath/functions.py]: There are some tabs here. Please remove them

**Review by mambelli — CHANGES_REQUESTED** [review]

File: src/planetsmath/functions.py (hunk 1 above)
Comment (GuillermoFidalgo) [reply src/planetsmath/functions.py]: Should be fixed now!

File: src/planetsmath/functions.py [hunk 2]
```diff
@@ -28,3 +28,11 @@ def sum_product(list):
...
         product *= item
     return product
+
+
+def make_author_print(author="Guillermo"):
+    """This function makes a print of the author of this change
+    This should be made with spaces now
+    """
+    print("Guillermo Made a fork")
+
```
Comment (mambelli) [inline src/planetsmath/functions.py]: Remove this blank line

**Review by mambelli — CHANGES_REQUESTED** [review]

**Review by mambelli — APPROVED** [review]

**Pull request merged**